### PR TITLE
feat: code restructured

### PR DESCRIPTION
**Pull request by dev-sankhadip**
#24  code restructured done

### CODE REVIEW

**Comment by xadahiya** [comment]
Why do we have commented out code? Also please specify what did you change and how it improves the overall structure

**Comment by dev-sankhadip** [comment]
Every bit of server code is written in one file app.py. So, i have separated those views in another file **func.py** inside **lib** dir. I have created views in that file and have imported them inside app.py. And using add_url_rule i have mapped views to particular url. I have commented out codes in app.py because if there is any problem you found in codebase then wen will get the code instantly.

**Comment by Mec-iS** [comment]
* change the name from `func.py` to `utils.py`
* what are all the lines commented in the new version?

**Comment by dev-sankhadip** [comment]
Those are the codes which have been moved in func.py. I have commented out in case if any error i can get those from that file.

**Comment by Mec-iS** [comment]
Please get [more familiar with git](https://hackernoon.com/understanding-git-fcffd87c15a3). If your changes break something you can just unroll them by reversing a commit.
The git data structure is a tree that tracks all the changes made since initialisation of the repository, this way you can track and navigate all the code removed/added. When you are not sure of your changes, you can just remove the commit and get to the previous state of the code.

**Comment by dev-sankhadip** [comment]
Thanks @Mec-iS,  i will look upon it. Please check if it is correct now

**Comment by dev-sankhadip** [comment]
Any update?

File: app.py [hunk 1]
```diff
@@ -5,6 +5,10 @@
 import json, os
 from requests import get
 
+from lib.func import serve, start_agent, hydra_doc, apidoc_graph, send_command, default
```
Comment (sameshl) [inline app.py]: When you have renamed `func.py` to `utils.py`, I think you need to fix the imports here from `lib.func` to `lib.utils`.

File: app.py [hunk 2]
```diff
@@ -14,196 +18,12 @@
...
-            return "Delete request must contain a url"
-        return json.dumps(agent.delete(url))
-    else:
-        return "Method not supported."
+app.add_url_rule('/<path:path>','serve',serve)
+app.add_url_rule('/start-agent','start-agent', start_agent)
+app.add_url_rule('/hydra-doc','hydra-doc', hydra_doc)
+app.add_url_rule('/apidoc-graph','apidoc-graph', apidoc_graph)
+app.add_url_rule('/send-command','send-command', send_command)
+app.add_url_rule('/','default',default)
```
Comment (sameshl) [inline app.py]: You should try to follow PEP8 convention here. Try to give a space after every comma in the function parameter list. 
`app.add_url_rule('/<path:path>','serve',serve)`  -> `app.add_url_rule('/<path:path>', 'serve', serve)`

**Comment by dev-sankhadip** [comment]
Updated

**Comment by dev-sankhadip** [comment]
Please check my pull request.

**Pull request closed without merging**